lib/services/ecf/payment_calculation.rb

```diff
@@ -10,7 +10,7 @@ def call
           input: config.to_h,
           output: {
             service_fees: ServiceFees.call(config),
-            variable_payments: VariablePaymentsAggregator.call(config),
+            output_payments: OutputPaymentAggregator.call(config),
```

Comment:
This should be singular too. It's still a payment to a provider, even though there are per participant values as part of the breakdown and a "payment over time" element.